lib/tasks/import_datastream.rake

```diff
@@ -22,23 +21,22 @@ namespace :ssg do
         data: OpenshiftEnvironment.summary
       )
     end
+    Rails.cache.delete('latest_supported_benchmarks')
   end
 
   desc 'Update compliance DB with the supported SCAP Security Guide versions'
   task import_rhel_supported: [:environment] do
     # DATASTREAM_FILENAMES from openscap_parser's ssg:sync
-    begin
```

Comment:
I removed this block, since there was no `rescue` as @bastilian pointed out.

Reply:
Not sure why rubocop did not complained. :smile: